src/VisionModel/resnet.jl

```diff
@@ -0,0 +1,79 @@
+using Flux
+
+function basicblock(inplanes::Int, planes::Int, stride::Int, dilation::Int)
+  basic =  Chain(
+      Conv((3, 3), inplanes=>planes, stride=stride, pad=(1,1), dilation=dilation),
+      BatchNorm(planes, λ = relu),
+      Conv((3, 3), planes=>planes, stride=stride, pad=(1,1), dilation=dilation),
+      BatchNorm(planes, λ=relu)
+    );
+  return basic
+end
+
+function bottleneck(inplanes::Int, planes::Int)
+  base_width = 64;
+  groups = 1;
+  width = Int(planes * (base_width / 64.)) * groups;
+  residual =  Chain(Conv((1, 1), inplanes=>width, stride=(2,2)),
+          BatchNorm(width, λ=relu),
+          Conv((3, 3), width=>width, stride=(2,2), pad=(1,1), dilation=dilation),
+          BatchNorm(width, λ=relu),
+          Conv((1, 1), width=>planes * expansion, stride=(2,2)),
+          BatchNorm(planes * expansion, λ=relu));
+  return residual
+end 
+
+function projection(inplanes::Int, planes::Int, stride::Int, expansion::Int)
```

Comment:
Can you convert this to the `inplanes => outplanes` style instead of using `expansion`?